src/google/protobuf/descriptor.cc

```diff
@@ -7675,6 +7676,14 @@ bool DescriptorBuilder::OptionInterpreter::ExamineIfOptionIsSet(
   return true;
 }
 
+#define VALUE_OUT_OF_RANGE(T, NAME) absl::StrFormat(  \
```

Comment:
Please us a templated helper function that passes in the type name expicitly.  macros are a thing we usually avoid for maintence reasons (they interfere with static analysis

Reply:
Roger that.

Reply:
Fixed